backend/internal/common/env_config.go

```diff
@@ -53,6 +54,7 @@ var EnvConfig = &EnvConfigSchema{
 	Port:               "1411",
 	Host:               "0.0.0.0",
 	UnixSocket:         "",
+	UnixSocketMode:     "0666",
```

Comment:
Should default value be empty? So we avoid running the chmod unless the user explicitly wants it?